pkg/workload/schemachange/optype.go

```diff
@@ -284,15 +330,15 @@ var opWeights = []int{
 	createIndex:                       1,
 	createSchema:                      1,
 	createSequence:                    1,
-	createTable:                       1,
+	createTable:                       500,
```

Comment:
kindly ignore the value of this right now, but wdyt of cranking this sucker up to 10?